docs/README.md

```diff
@@ -917,8 +917,8 @@ General API HTTP status codes and what to expect of them.
 
 | Field         | Type                      | Required           | Example                              | Description                                                                                                                                                                |
 | ------------- | ------------------------- | ------------------ | ------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| unitPrice     | integer                   | <center>x</center> | 1000                                 | Price per unit, in each country's minor unit, e.g. for Euros use cents. By default price should include VAT, unless [`usePricesWithoutVat`](#request-body) is set to true. |
-| units         | integer                   | <center>x</center> | 5                                    | Quantity, how many items ordered                                                                                                                                           |
+| unitPrice     | integer                   | <center>x</center> | 1000                                 | Price per unit, in each country's minor unit, e.g. for Euros use cents. By default price should include VAT, unless [`usePricesWithoutVat`](#request-body) is set to true. Only positive values accepted. |
+| units         | integer                   | <center>x</center> | 5                                    | Quantity, how many items ordered. Only positive values supported.                                                                                                                                          |
```

Comment:
Is zero okay here?

Reply:
Actually, in some cases zero can be used: if the total amount of all items is more than zero, then some of the items can be set to zero.